web/job_specs_controller_test.go

```diff
@@ -32,27 +34,51 @@ func TestJobSpecsController_Index(t *testing.T) {
 	app, cleanup := cltest.NewApplication()
 	defer cleanup()
 
-	j1 := setupJobSpecsControllerIndex(app)
+	j1, err := setupJobSpecsControllerIndex(app)
+	assert.NoError(t, err)
 
-	resp := cltest.BasicAuthGet(app.Server.URL + "/v2/specs")
-	assert.Equal(t, 200, resp.StatusCode, "Response should be successful")
+	resp := cltest.BasicAuthGet(app.Server.URL + "/v2/specs?size=x")
+	cltest.AssertServerResponse(t, resp, 422)
+
+	resp = cltest.BasicAuthGet(app.Server.URL + "/v2/specs?size=1")
+	cltest.AssertServerResponse(t, resp, 200)
+
+	var links jsonapi.Links
+	jobs := []models.JobSpec{}
 
-	var jobs []models.JobSpec
-	json.Unmarshal(cltest.ParseResponseBody(resp), &jobs)
+	err = web.ParsePaginatedResponse(cltest.ParseResponseBody(resp), &jobs, &links)
+	assert.NoError(t, err)
+	assert.NotEmpty(t, links["next"].Href)
```

Comment:
Seems worth checking what the response is, like maybe just a regex that checks size is persisted and page is added/incremented?

Reply:
This is already tested by the `api_test.go` so I feel like adding coverage here might be a bit redundant?